docs/_data/repoStatus.yaml

```diff
@@ -155,13 +164,23 @@
     docs: ready
 - tagName: rh-jump-links
   name: Jump links
-  type: element
+  type: coming-soon
```

Comment:
We have planned for this, better to keep it there

Reply:
Resolved in [cc1b616](https://github.com/RedHat-UX/red-hat-design-system/pull/1988/commits/cc1b6164c2e4206325c9cbc4959bba381bbe73b1).

Reply:
@bennypowers did you just want `rh-jump-links` marked as `coming-soon` rather the whole thing reverted?